kaspresso/src/main/kotlin/com/kaspersky/kaspresso/instrumental/InstrumentalUtils.kt

```diff
@@ -0,0 +1,13 @@
+package com.kaspersky.kaspresso.instrumental
+
+import androidx.test.platform.app.InstrumentationRegistry
+
+/**
+ * @return true if an environment is instrumental, or false if the environment is JVM (with Robolectric)
+ */
+internal fun checkInstrumentalStateOfEnvironment(): Boolean =
+    try {
+        InstrumentationRegistry.getInstrumentation().uiAutomation != null
+    } catch(exception: Exception) {
```

Comment:
Maybe better to catch only a more precise Exception, like `NullPointerException`?
In doing so, we can avoid swallowing other undesired exceptions.